### PR TITLE
Create 2023.09.05_meeting_notes.md

### LINKED ISSUE #261: OpenJS Foundation Standards Working Group Meeting 2023-09-05
## Time

**UTC Tue 05-Sep-2023 17:00 (05:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Tue 05-Sep-2023 10:00 (10:00 AM) |
| US / Mountain | Tue 05-Sep-2023 11:00 (11:00 AM) |
| US / Central  | Tue 05-Sep-2023 12:00 (12:00 PM) |
| US / Eastern  | Tue 05-Sep-2023 13:00 (01:00 PM) |
| EU / Western  | Tue 05-Sep-2023 18:00 (06:00 PM) |
| EU / Central  | Tue 05-Sep-2023 19:00 (07:00 PM) |
| EU / Eastern  | Tue 05-Sep-2023 20:00 (08:00 PM) |
| Moscow        | Tue 05-Sep-2023 20:00 (08:00 PM) |
| Chennai       | Tue 05-Sep-2023 22:30 (10:30 PM) |
| Hangzhou      | Wed 06-Sep-2023 01:00 (01:00 AM) |
| Tokyo         | Wed 06-Sep-2023 02:00 (02:00 AM) |
| Sydney        | Wed 06-Sep-2023 03:00 (03:00 AM) |

Or in your local time:
* https://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Standards%20Working%20Group+Meeting+2023-09-05&iso=20230905T1700
* or https://www.wolframalpha.com/input/?i=05PM+UTC%2C+Sep+05%2C+2023+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1R61EMtcCfAEVtfKTrDdNzsaCb9sIROzrtwBGxMA3AeQ/edit>

## Agenda

Extracted from **standards-agenda** labelled issues and pull requests from the **openjs-foundation org** prior to the meeting.

### openjs-foundation/standards

* W3C wants to increase the size of the TAG [#260](https://github.com/openjs-foundation/standards/issues/260)
* 2024 Meeting / Schedule Conflicts [#259](https://github.com/openjs-foundation/standards/issues/259)
* Document Guidance for requesting travel funds for Standards WG-related activities [#256](https://github.com/openjs-foundation/standards/issues/256)
* Check Standards WG agrees to migrate to a Collab Space [#253](https://github.com/openjs-foundation/standards/issues/253)
* WIP - Improvements toward Addressing #173 [#240](https://github.com/openjs-foundation/standards/pull/240)
* Update MEMBER_REPRESENTATION.md [#239](https://github.com/openjs-foundation/standards/pull/239)
* Define terms for SME Participation [#173](https://github.com/openjs-foundation/standards/issues/173)

## Invited

* OpenJS Foundation Cross Project Council
* OpenJS Foundation Project Maintainers
* OpenJS Foundation Board of Directors

## Notes

The agenda comes from issues labelled with `standards-agenda` across **all of the repositories in the openjs-foundation org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

link for participants: Zoom link: <https://zoom.us/j/599005386>

* For those who just want to watch: <https://livestream.openjsf.org>

---

**Invitees**

Please use the following emoji reactions in this post to indicate your
availability.

* :+1: - Attending
* :-1: - Not attending
* :confused: - Not sure yet

### PR DESCRIPTION
**Pull request by jorydotcom**
Closes #261

### CODE REVIEW

**Review by ljharb — APPROVED** [review]

File: meeting-notes/2023.09.05_meeting_notes.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+* Jory Burson @jorydotcom 
+* Tobie Langel @tobie
+* Joe Sepi @joesepi
+* Rick Markins @rxmarbles
+* Eemeli Aro @eemeli
+* Mike Samuel @mikesamuel
+* Jordan Harband @ljharb
+* Richard Gibson @gibson042
+* Lea Verou @leaverou
```
Comment (VoltrexKeyva) [inline meeting-notes/2023.09.05_meeting_notes.md:18]:
```suggestion
* Jory Burson (@jorydotcom)
* Tobie Langel (@tobie)
* Joe Sepi (@joesepi)
* Rick Markins (@rxmarbles)
* Eemeli Aro (@eemeli)
* Mike Samuel (@mikesamuel)
* Jordan Harband (@ljharb)
* Richard Gibson (@gibson042)
* Lea Verou (@leaverou)
```
To be consistent with the previous meeting notes.

File: meeting-notes/2023.09.05_meeting_notes.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+## Agenda
+
+### Announcements
+
+*Extracted from **cross-project-council-agenda** labeled issues and pull requests from the **openjs-foundation org** prior to the meeting.
```
Comment (VoltrexKeyva) [inline meeting-notes/2023.09.05_meeting_notes.md:24]:
```suggestion
* Extracted from **cross-project-council-agenda** labeled issues and pull requests from the **openjs-foundation org** prior to the meeting.
```

File: meeting-notes/2023.09.05_meeting_notes.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+* [DOM Localization](https://github.com/w3c/tpac2023-breakouts/issues/52)
+* [The Future of Powerful APIs on the Web Platform](https://github.com/w3c/tpac2023-breakouts/issues/76)
+
+Travel Fund progress continues - follow CPC meetings and WG sessions for more.
+
+### openjs-foundation/standards
+
+* W3C wants to increase the size of the TAG [#260](https://github.com/openjs-foundation/standards/issues/260)
+Lea brought the issue with others; Tobie submitted our support. 
+1 elected, 1 appointed. Would likely happen during next election cycle.
```
Comment (VoltrexKeyva) [inline meeting-notes/2023.09.05_meeting_notes.md:36]:
```suggestion
Lea brought up the issue with others; Tobie submitted our support. 
1 elected, 1 appointed. Would likely happen during the next election cycle. 
```

File: meeting-notes/2023.09.05_meeting_notes.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+### openjs-foundation/standards
+
+* W3C wants to increase the size of the TAG [#260](https://github.com/openjs-foundation/standards/issues/260)
+Lea brought the issue with others; Tobie submitted our support. 
+1 elected, 1 appointed. Would likely happen during next election cycle. 
+Vote closes today (Sept. 5) 
+
+* 2024 Meeting / Schedule Conflicts [#259](https://github.com/openjs-foundation/standards/issues/259)
+Jory to update public calendar as dates are locked in
```
Comment (VoltrexKeyva) [inline meeting-notes/2023.09.05_meeting_notes.md:40]:
```suggestion
Jory to update the public calendar as dates are locked in
```

File: meeting-notes/2023.09.05_meeting_notes.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+* Document Guidance for requesting travel funds for Standards WG-related activities [#256](https://github.com/openjs-foundation/standards/issues/256)
+CPC is looking to create a rubric for scoring.
+https://github.com/openjs-foundation/cross-project-council/issues/1154 
+Action: provide any feedback / comments to #1154
```
Comment (VoltrexKeyva) [inline meeting-notes/2023.09.05_meeting_notes.md:45]:
```suggestion
Action: provide any feedback/comments to #1154
```

File: meeting-notes/2023.09.05_meeting_notes.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+Related: what are the P0s for Standards Related travel in 2024? How can we tie the impact of this work to specific 
+
+* Check Standards WG agrees to migrate to a Collab Space [#253](https://github.com/openjs-foundation/standards/issues/253)
+No further discussion needed
+We need a charter, small tasks - CPC will let us know what next.
```
Comment (VoltrexKeyva) [inline meeting-notes/2023.09.05_meeting_notes.md:50]:
```suggestion
We need a charter and small tasks - CPC will let us know what next.
```

File: meeting-notes/2023.09.05_meeting_notes.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+Comments to be addressed and PR merged
+
+* Update MEMBER_REPRESENTATION.md [#239](https://github.com/openjs-foundation/standards/pull/239)
+Conflict to be resolved and merged
+
+* Define terms for SME Participation [#173](https://github.com/openjs-foundation/standards/issues/173)
+
+AOB 
+Mike Samuel asked in chat RE language translations and thread models in chat. Brief discussion ensued about which organizations are keeping this in mind. 
+MessageFormat spec update from Eemeli - feels like this spec will be out spring 2024
```
Comment (VoltrexKeyva) [inline meeting-notes/2023.09.05_meeting_notes.md:62]:
```suggestion
Mike Samuel asked in chat about language translations and thread models. A brief discussion ensued about which organizations are keeping this in mind. 
MessageFormat spec update from Eemeli - feels like this spec will be out in spring 2024
```

**Pull request merged**